fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/BrewingRecipeRegistryAccessor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import java.util.List;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import net.minecraft.item.Item;
+import net.minecraft.potion.Potion;
+import net.minecraft.recipe.BrewingRecipeRegistry;
+
+@Mixin(BrewingRecipeRegistry.class)
+public interface BrewingRecipeRegistryAccessor {
```

Comment:
Since there's already an AW file in the module, you can switch that accessor mixin for an AW.

Reply:
Is there a contribution guideline for this that I've missed?
Otherwise we're just getting into the access wideners vs accessor mixins debate.

Reply:
Is there a contribution guideline for this that I've missed?

Reply:
AWs are cleaner than mixins, however they require the module to have its own jar, which makes the build slower. But if there's already an AW for other reasons then it's better to use the AW (less class spam). This isn't in the contribution guidelines though I think, maybe they should be amended?